libs/shared/components/README.md

```diff
@@ -0,0 +1,7 @@
+# shared-components
```

Comment:
recreate the lib as "dynamic-form" outside of shared (shared is for things that are also used in the daemon)
im also suggesting dynamic-form instead of components because this lib will also have custom fields, validators, etc